ui/packages/image/src/SketchSettings.svelte

```diff
@@ -22,9 +22,14 @@
 	<!-- <IconButton Icon={Undo} on:click={() => dispatch("undo")} /> -->
 
 	<span class="absolute top-0 right-0">
-		<IconButton Icon={Brush} on:click={() => (show_size = !show_size)} />
+		<IconButton
+			Icon={Brush}
+			aria-label="Use brush"
+			on:click={() => (show_size = !show_size)}
+		/>
 		{#if show_size}
 			<input
+				aria-label="Brush radius"
```

Comment:
`label`

Reply:
I think this comment's on the wrong line, the <input> will need to keep aria-label but I updated the <IconButton> prop line.